.travis.yml

```diff
@@ -25,6 +25,7 @@ install: source continuous_integration/travis/install.sh
 script:
   - source continuous_integration/travis/run_tests.sh
   - flake8 dask
+  - pycodestyle --select=E722 --exclude=tests dask  # check for bare `except:` blocks
```

Comment:
It could in this PR, but I don't think we need to be overly strict about tests going forward.  Happy to remove `--exclude=tests` if you prefer.